figures/pagination.py

```diff
@@ -1,13 +1,11 @@
-'''Paginatiors for Figures
-
+'''
```

Comment:
@OmarIthawi Might as well change to double quotes while you're modifying this

Reply:
@OmarIthawi Please see https://www.python.org/dev/peps/pep-0008/#documentation-strings

Reply:
:eyes: 

I see PEP-0008, but I don't like what I see.

I'll just keep it as-is!

Reply:
Reverted.